### PR TITLE
[DC-1799] Converted retraction dates to timestamps

### PR DESCRIPTION
**Pull request by jp3477**

### CODE REVIEW

File: data_steward/retraction/retract_deactivated_pids.py
Comment (hiro-mishima) [inline data_steward/retraction/retract_deactivated_pids.py:52]: Please update the column name from `deactivated_date` to `deactivated_datetime`, as @molly-giddings's DC-1795 will be also updating the column name.

**Review by hiro-mishima — CHANGES_REQUESTED** [review]
Thanks for working on this, @jp3477!
I added one comment for changes. Can you please check it out? The integration test will NOT likely pass still, as this development is dependent on DC-1795. We can work on that once DC-1795 is almost complete.

File: data_steward/retraction/retract_deactivated_pids.py
Comment (jp3477) [reply data_steward/retraction/retract_deactivated_pids.py:52]: @hiro-mishima Thanks for cross-referencing these two tickets. Do you think it would be best to hold off on the final work for this until DC-1795 is merged?

**Comment by hiro-mishima** [comment]
Closing this PR as this change was included in DC-1795. Thanks!

**Pull request closed without merging**
